pkgs/development/tools/database/prisma-engines/default.nix

```diff
@@ -14,13 +14,13 @@
 # function correctly.
```

Comment:
Oops, missed this.

Reply:
All this `nodePackages` stuff is a pain to update. It might be easier to move `prisma` into a regular package.

Reply:
Possible to update the nodePackages.prisma, as well, or keep the same version of engines and just modify the cargo lock with updated times dependency?

Reply:
I second this: with this current fix, `nodePackages.prisma` becomes unbuildable because of a wrong hash in `pkgs/development/node-packages/overrides.nix`. Setting the good hash allow me to build `nodePackages.prisma` and to update `pingvin-share`, which is based on prisma

Reply:
> All this nodePackages stuff is a pain to update. It might be easier to move prisma into a regular package.

Unfortunately it's not easy at all (because of pnpm) — but it needs to be done anyway indeed.

Reply:
Nixpkgs does have pnpm support now though. I'll look into it.

Reply:
Yeah, pnpm packages are actually simple to implement, now.